clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus.go

```diff
@@ -265,6 +310,8 @@ func (m *apiCallMetrics) ToPerfData() *measurementutil.PerfData {
 				"Subresource": apicall.Subresource,
 				"Scope":       apicall.Scope,
 				"Count":       fmt.Sprintf("%v", apicall.Count),
+				// FailedCount should be considered implementation details
+				// and there no need to include it in PerfData
```

Comment:
I disagree - this may affect whether the measurement fails or not, so I would like to see that in summary.

Reply:
Added.